src/styles/toolbar.styl

```diff
@@ -6,6 +6,18 @@
     .coz-btn:last-child
         margin-right 0
 
+.fil-toolbar-trash
+    @extends $buttons
```

Comment:
@GoOz  and @m4dz : this one was an interesting problem. The related DOM is [here](https://github.com/y-lohse/cozy-files-v3/blob/FIL-20/src/containers/TrashToolbar.jsx#L15).

The problem I encountered was, the first button (`coz-btn--delete`) really needs the `danger-outline` class that is defined [here](https://github.com/cozy/cozy-ui/blob/v3/stylus/ui-components/button.styl#L118). But when I do that, I import the selector `[role=button], button`, and now [the other button](https://github.com/y-lohse/cozy-files-v3/blob/FIL-20/src/containers/TrashToolbar.jsx#L21) *also* inherits these styles -- because this selector is stronger than `coz-btn coz-btn--more`.

Is there a way to @extend only some parts of the placeholder? That would be quite handy in cases like this.